webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -50,10 +59,10 @@ class ManageManifestModal extends Component {
   }
 
   uploadManifest(fileList) {
+    this.setState({ actionInProgress: true });
     if (fileList.length > 0) {
-      this.props.uploadManifest(fileList[0]).then(this.props.loadOrganization);
```

Comment:
uploadManifest creates an asynchronous  task, so I moved the loadOragnization to `componentDidUpdate`